Automattic/wp-calypso
# Pattern Library: Add icon to copy button

Fixes https://github.com/Automattic/dotcom-forge/issues/6424

## Proposed Changes

This PR adds a `copy` icon to the copy buttons for patterns. The screencasts below shows it in action:

| List | Grid |
| - | - |
| <video src="https://github.com/Automattic/wp-calypso/assets/1101677/c89565ae-6a48-4667-ba76-60a99940bb3e"></video> | <video src="https://github.com/Automattic/wp-calypso/assets/1101677/80cd1576-a83c-4b03-af17-45bfe907724b"></video> |


## Testing Instructions



1. Navigate to `/patterns/about`
2. Ensure that the copy button looks good
3. Click the copy button
4. Ensure that it looks good once it changes to the `check` icon too
5. Repeat the same process in grid view
